rc/extra/nim.kak

```diff
@@ -53,9 +53,37 @@ add-highlighter shared/nim/code regex %{'[^'\n]'} 0:string
 
 add-highlighter shared/nim/code regex '#[^\n]+' 0:comment
 
+# Commands
+# ‾‾‾‾‾‾‾‾
+
+def -hidden nim-indent-on-new-line %{
+    eval -draft -itersel %{
+        # copy '#' comment prefix and following white spaces
+        try %{ exec -draft k <a-x> s ^\h*#\h* <ret> y jgh P }
+        # preserve previous line indent
+        try %{ exec -draft \; K <a-&> }
+        # cleanup trailing whitespaces from previous line
+        try %{ exec -draft k <a-x> s \h+$ <ret> d }
+        # indent after line ending with const, let, var, ':' or '='
+        try %{ exec -draft <space> k x <a-k> (:|=|const|let|var)$ <ret> j <a-gt> }
+    }
+}
+
 # Initialization
 # ‾‾‾‾‾‾‾‾‾‾‾‾‾‾
 
 hook -group nim-highlight global WinSetOption filetype=nim %{ add-highlighter window ref nim }
 
+hook global WinSetOption filetype=nim %{
+    set buffer tabstop 2
+    set buffer indentwidth 2
```

Comment:
Please remove those two lines, users are free to chose whatever indentation they want to edit nim code.

Reply:
yeah, looking at other langs, they dont have this set.
its just i was looking at pony's script and it had indentation settings.
removed.

Reply:
Alright, they're gone now #1764.